### DISCUSSION TITLE
GitHub asked to verify my email, my account is almost a year old

### DISCUSSION THREAD

**Discussion by trentwiles** [community/community#22171] [New to GitHub]
<p>Got a bit of an odd message tonight:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/6/b689c8caaab03025c3c9bbb818b4649a292f14ae.png" data-download-href="/uploads/short-url/q2O6h47CSV9RqH0WOygFAsh4VbM.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59586759/181069319-639ab820-7abb-45fa-88d0-001e861a2aa0.png" alt="" data-base62-sha1="q2O6h47CSV9RqH0WOygFAsh4VbM" width="690" height="80" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b689c8caaab03025c3c9bbb818b4649a292f14ae_2_690x80.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b689c8caaab03025c3c9bbb818b4649a292f14ae_2_1035x120.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b689c8caaab03025c3c9bbb818b4649a292f14ae_2_1380x160.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b689c8caaab03025c3c9bbb818b4649a292f14ae_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename"></span><span class="informations">1920×224 26.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Github out of the blue wanted to verify my email. Is this a normal thing for security reasons? I even got an email about it:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/b/8bddc3ba92bc6ff28f330191c46f5255c575f4a0.png" data-download-href="/uploads/short-url/jXjuNDiJtugesoErddOO4Zzd51C.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59586759/181069320-85c401d0-ddb8-4dd5-a726-9f45de2ccb5e.png" alt="" data-base62-sha1="jXjuNDiJtugesoErddOO4Zzd51C" width="690" height="360" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8bddc3ba92bc6ff28f330191c46f5255c575f4a0_2_690x360.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8bddc3ba92bc6ff28f330191c46f5255c575f4a0_2_1035x540.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/b/8bddc3ba92bc6ff28f330191c46f5255c575f4a0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/b/8bddc3ba92bc6ff28f330191c46f5255c575f4a0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename"></span><span class="informations">1109×579 33.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>After verifying my email, I was prompted to set up my account:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/3/23d6fba99bbf437c92671a90962e989e5613af82.png" data-download-href="/uploads/short-url/573jfIw0abcX9SuugWpyxbHlDoK.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59586759/181069321-7f4b68dc-0182-44ee-933f-5c4e14260468.png" alt="" data-base62-sha1="573jfIw0abcX9SuugWpyxbHlDoK" width="690" height="314" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/3/23d6fba99bbf437c92671a90962e989e5613af82_2_690x314.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/3/23d6fba99bbf437c92671a90962e989e5613af82_2_1035x471.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/3/23d6fba99bbf437c92671a90962e989e5613af82_2_1380x628.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/3/23d6fba99bbf437c92671a90962e989e5613af82_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename"></span><span class="informations">1920×876 162 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Is this a bug?</p>

**Comment by david-driscoll** [comment]
<p>I’ve actually had the same thing happen.  There was the google outage yesterday, which I figured is what caused it.  However it happened again today.</p>
<p>So 2 days in a row I’ve had to reverify my email, I’ve had a github account for 6 or 7 years or something like that at this point.</p>
<p>It is very strange.</p>

**Comment by manishlok** [comment]
<p>Hi,<br>
I am getting alert to verify email every time I login to github. I had verified my email address when I signed up to github during June,2018 .<br>
But from yesterday, whenever I login to github , I got alert to verify email. I verified my email 2 times from yesteerday. Please see the issue.</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/riversiderocks">@RiversideRocks</a>,</p>
<p>Thank you for being here and reaching out about this. Our teams have been looking into this from our end. This issue looks to be related to an on-going (and somewhat intermittent) issue that Google is experiencing with Gmail addresses. <a href="https://www.google.com/appsstatus#hl=en&amp;v=status">Google’s own dashboard</a> can be found here if you have further interest in this issue.</p>
<p>From GitHub’s side, for security reasons we’ll un-verify any email address that is bounced back to us; to both increase awareness of delivery issues and help ensure important system emails (like password resets) are capable of reaching our users when needed. <em>Our team is looking into alternate ways to assist our users with this issue.</em>  In the interim:</p>
<ul>
<li>
<p>You should be able to perform a re-verification of your address either directly from <a href="https://github.com/settings/emails">your account’s email settings</a></p>
</li>
<li>
<p>Or by following the steps outlined in this <a href="https://docs.github.com/articles/verifying-your-email-address">support article</a></p>
</li>
</ul>

**Comment by joerick** [comment]
<p>I also had this issue. Turns out I had set up a forwarder on my email account, because I wanted to try out a new mail service (<a href="http://hey.com" rel="noopener nofollow ugc">hey.com</a>). When the <a href="http://Hey.com" rel="noopener nofollow ugc">Hey.com</a> trial ended, they started bouncing mail that was sent to them, and the bounces went back to Github, so Github kept on assuming the email address was bad.</p>
<p>Disabling the mail forwarder fixed it. Maybe a few other people have the same situation. It’s very strange, because a few other websites, like Linkedin also started to report that my email address was invalid, took me ages to figure it out!</p>

**Comment by canuckjacq** [comment]
<p><a class="mention" href="/manishlok">@manishlok</a></p>
<p>:wave: Welcome!</p>
<p>We did have a lot of people writing in after the global Gmail outage. If your email account was down, and any emails from us bounced, you will have to re-verify.</p>
<p>If that isn’t the case, you may need to speak to whomever it is that administrates your email server. If they have set up new rules that have resulted in emails from GitHub bouncing, for instance, that can affect it too.</p>
